### DISCUSSION TITLE
Handling Input Errors

### DISCUSSION THREAD

**Discussion by MitchellGriggsTAFE** [MitchellGriggsTAFE/TafeCalc24#3] [Ideas]
I noticed there's nothing to theoretically stop a user from inputting non-numerical values to the calculators. Would it be prudent to display an error message in this case and prompt the user to re-enter their input?

**Comment by TravisGriggsProjects** [comment]
Yes, absolutely, we should always be on-guard against potential oversights in our applications. I'll be sure to include an error message in the Mortgage Calculator as well.
